<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-E5GIE34" failures="1" tests="3" name="TestNG_Learning.TestCase3_DependsOn" time="0.076" errors="0" timestamp="2022-05-07T20:40:51 IST" skipped="2">
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="doUserReg3" time="0.072">
    <failure message="Registration failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Registration failed
at org.testng.Assert.fail(Assert.java:99)
at TestNG_Learning.TestCase3_DependsOn.doUserReg3(TestCase3_DependsOn.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- doUserReg3 -->
  <system-out>
    <![CDATA[<a href='..\screenshots\Golf Clubsshot2.png' target='_blank'>ScreenShot </a>
</br><a href='..\screenshots\Golf Clubsshot2.png' target='_blank'><img src='..\screenshots\Golf Clubsshot2.png' height=400 width=400> </a>]]>
  </system-out>
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="doLogin3" time="0.000">
    <skipped/>
  </testcase> <!-- doLogin3 -->
  <system-out/>
  <testcase classname="TestNG_Learning.TestCase3_DependsOn" name="isSkip" time="0.004">
    <skipped/>
  </testcase> <!-- isSkip -->
  <system-out/>
</testsuite> <!-- TestNG_Learning.TestCase3_DependsOn -->
